aries-core/src/js/components/external/stories/Tabs.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import {
+  Box,
+  Grommet,
+  Tab,
+  Tabs,
+  Table,
+  TableBody,
+  TableRow,
+  TableCell,
+  Text,
+} from 'grommet';
+import { aries } from '../../../../../../aries-site/src/themes/aries';
+
+export default {
+  title: 'Tabs',
+};
+
+export const Simple = () => {
+  const [index, setIndex] = useState();
+  const onActive = nextIndex => setIndex(nextIndex);
+
+  return (
+    <Grommet theme={aries} full>
```

Comment:
very long code example, we should break it down to internal components, each Tab should have ~1-4 child lines, so I'll extract for example the Table to a separate component.

Reply:
Good clean up idea. I'll implement it.